Editor/Window/ManagedEC2Deployment.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEngine;
+
+namespace AmazonGameLift.Editor
+{
+    public class ManagedEC2Deployment
+    {
+        private readonly DeploymentSettings _deploymentSettings;
+        
+        internal ManagedEC2Deployment(DeploymentSettings deploymentSettings, ManagedEC2FleetParameters parameters)
+        {
+            _deploymentSettings = deploymentSettings;
+            UpdateModelFromParameters(parameters);
+        }
+        
+        public void UpdateModelFromParameters(ManagedEC2FleetParameters parameters)
+        {
+            _deploymentSettings.FleetName = parameters.FleetName;
+            _deploymentSettings.BuildName = parameters.BuildName;
+            _deploymentSettings.LaunchParameters = parameters.LaunchParameters;
+            _deploymentSettings.BuildFolderPath = parameters.GameServerFolder;
+            _deploymentSettings.BuildFilePath = parameters.GameServerFile;
+            _deploymentSettings.BuildOperatingSystem = parameters.OperatingSystem;
+        }
+        
+        public void StartDeployment()
+        {
+            _deploymentSettings.GameName = Application.productName.Substring(0, 12);
```

Comment:
This comment was probably lost elsewhere. If the productName can be modified by the user, this assumption becomes invalid.

Reply:
What's the solution then? There is no input in the mock for this

Reply:
We could also just use the default from the CF template, currently `MyGame` and make the user update that if they require

Reply:
I see. And this was a parameter in the UX previous prior to your changes? We should ask Chelsey on Slack. This is used in the CF template, so we should confirm if that is the end user experience they are ok with.